_security-analytics/api-tools/threat-intel/threat-intel-source.md

```diff
@@ -0,0 +1,637 @@
+---
+layout: default
+title: Threat Intelligence Source Configuration APIs
```

Comment:
```suggestion
title: Threat intelligence source APIs
```
Can we shorten this to Threat intelligence source APIs or Thread intelligence APIs? See the other headings in this section.